notebooks/community/ml_ops/stage7/model_monitoring_custom.ipynb

```diff
@@ -0,0 +1,1267 @@
+{
```

Comment:
<p>Can we show how the monitoring job is useful here? Like adding some MM job results and how to interpret them? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Reply:
<p>I struggled with that. The monitoring job has to run a min of one hour before the results can be analyzed. I looked the MM/blog notebook. The way Marc handled that is he added screenshots of previous monitoring results.</p>